magefile.go

```diff
@@ -70,27 +69,14 @@ func Test() error {
 	if err := sh.RunV("go", "test", "./..."); err != nil {
 		return err
 	}
+	if err := sh.RunV("go", "test", "./examples/http-server"); err != nil {
+		return err
+	}
+	if err := sh.RunV("go", "test", "./testing/coreruleset"); err != nil {
```

Comment:
This takes 15s on my laptop. It's not outrageous so I am leaning towards keeping it in the default test task, but let me know if there is more preference to having a separate task